src/components/comments/CommentList.js

```diff
@@ -1,28 +1,38 @@
-import React, { useEffect, useContext } from "react"
+import React, { useEffect, useContext, useState } from "react"
 import ListGroup from "react-bootstrap/ListGroup"
 import { CommentContext } from "./CommentProvider"
 import { Comment } from "./Comment"
 
 export const CommentList = props => {
-  const { postId } = props
-
   const { comments, getCommentsByPostId } = useContext(CommentContext)
-
+  console.log('Here', comments)
+  
   useEffect(() => {
+    const postId = parseInt(props.match.params.postId);
     getCommentsByPostId(postId)
   }, [])
 
   return (
-    <div className="commentList" style={{ maxHeight: '250px', overflowY: 'scroll' }}>
-      <ListGroup>
-        {
-          comments.map(c => (
-            <ListGroup.Item key={c.id}>
-              <Comment comment={c} />
-            </ListGroup.Item>
-          ))
-        }
-      </ListGroup>
+    <div>
+      <div className="commentList" style={{ maxHeight: '250px', overflowY: 'scroll' }}>
+        <ListGroup>
+
+          {
+            comments.map(c => (
+              <ListGroup.Item key={c.id}>
+                <Comment comment={c.content} />
+                <h1>subject: {c.subject}</h1>
+                <h4>{c.content}</h4>
+                <h4>{new Date(c.created_on).toLocaleDateString('en-Us')}</h4>
+                <h4>{c.user}</h4>
```

Comment:
This `c.user` isn't rendering anything right now, because we haven't sent the username in a comment from the server. Once you add that to the server-side comment, where the username can be sent in a serialized comment, I think changing this over to `c.username` should make all this work and actually render.